Tahi-project/tahi
# Aperta 13008

JIRA issue: https://jira.plos.org/jira/browse/APERTA-13008 and  https://jira.plos.org/jira/browse/APERTA-13011

#### What this PR does:

Adds very basic support for outputting regular (non-snapshot) cards to the export zip, as well as decisions and reviewer reports.

To test:

1. Check out this branch.
2. Load production data
3. Connect to prod S3
4. Run `rake export:random_manuscript_zips`
5. Check exported zips against ticket acceptance criteria
---

#### Code Review Tasks:

**Author tasks** (delete tasks that don't apply to your PR, this list should be finished before code review):

**Reviewer tasks** (these should be checked or somehow noted before passing on to PO):
